AUTHORS

```diff
@@ -53,17 +53,22 @@ Camilo Schoeningh                       camilo.schoeningh@dunnhumby.com
 Canh Ngo                                canhnt@gmail.com
 Carlos Castro                           carlos.castro@jumo.world
 Changyu Wang                            changyuwang@tencent.com
+Charles-Edouard Brétéché                charled.breteche@gmail.com
 Charles-Henri Guérin                    charles-henri.guerin@zenika.com
 chenyahui                               chenyahui9@jd.com
+chenyaqi01                              chenyaqi01@baidu.com
+chez-shanpu                             tomoki.sugiura@mail.shanpu.info
 Chris Tarazi                            chris@isovalent.com
 Christian Hörtnagl                      christian2@univie.ac.at
 Christian Hüning                        christian.huening@finleap.com
 Christine Chen                          christine.chen@datadoghq.com
 Christopher Biscardi                    chris@christopherbiscardi.com
 강주희                                  claudiajkang@gmail.com
+cndoit18                                cndoit18@outlook.com
 Connor Jones                            cj@cjmakes.com
 Craig Box                               craig.box@gmail.com
 Cynthia Thomas                          cynthia@covalent.io
+daemon1024                              barun1024@gmail.com
```

Comment:
Any chance for real names for these? The official policy for patch acceptance asks for this.

Reply:
for cndoit18 it was not possible